airbase/pom.xml

```diff
@@ -1126,18 +1127,16 @@
             </dependency>
 
             <!-- misc stuff -->
-
-            <!-- bval 1.1.1 depends on commons-lang3 3.3.2, which is incompatible with Java 9 -->
             <dependency>
```

Comment:
We can delete this dependency now. Based on the comment, it was added to force a newer version (than bval used) that is compatible with Java 9.